docs/ecosystem/officials/zio-mock.md

```diff
@@ -115,45 +115,45 @@ For example, to encode the `send` capability of `EmailService` we need to extend
 
 Let's see how we can mock the `EmailService`:
 
-```scala
+```scala mdoc:silent
 // Test Sources
+import zio._
 import zio.mock._
 
 object MockEmailService extends Mock[EmailService] {
   object Send extends Effect[(String, String), String, Unit]
 
   val compose: URLayer[Proxy, EmailService] =
-    ZIO
-      .service[Proxy]
-      .map { proxy =>
+    ZLayer {
+      for {
+         proxy <- ZIO.service[Proxy]
+      } yield (
```

Comment:
Very minor but this will be slightly more readable if you delete the parentheses here and define `new EmailService` on the same line. `} yield new EmailService {`